terraform.txt

```diff
@@ -1,15 +0,0 @@
-2019/12/10 10:04:26 [INFO] Terraform version: 0.12.17  
```

Comment:
What generated this file, and is it likely to be accidentally added again? Should it be in the .gitignore?

Reply:
I think it was Sam doing some debugging some time ago. Will add to git ignore in another PR